apps/web/src/components/SearchRequestTable/components/SearchRequestsTableFilterDialog.css

```diff
@@ -0,0 +1,8 @@
+/* Force modal to be near-full screen, regardless of breakpoint */
+[data-h2] #user-table-filter-dialog [data-reach-dialog-content] {
+  width: 95vw;
+}
```

Comment:
Is this doing anything? 🤔 We moved away from Reach UI so I don't believe that selector targets an actual element.

Reply:
Flat out copy-pasting from the first iteration of fancy filter. I'll remove it and see if anything changes.

Reply:
Removed in https://github.com/GCTC-NTGC/gc-digital-talent/pull/6897/commits/ee676d3b11948cbf1cd98c96953709e5ebd264b5